code/modules/mob/living/carbon/human/species.dm

```diff
@@ -930,25 +930,25 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 	bodyparts_to_add = handle_mutant_bodylist(human_host)
 
 	////PUT ALL YOUR WEIRD ASS REAL-LIMB HANDLING HERE
-	///Digi handling
-	if(human_host.dna.species.bodytype & BODYTYPE_DIGITIGRADE)
-		var/uniform_compatible = FALSE
-		var/suit_compatible = FALSE
-		if(!(human_host.w_uniform) || (human_host.w_uniform.supports_variations & DIGITIGRADE_VARIATION) || (human_host.w_uniform.supports_variations & DIGITIGRADE_VARIATION_NO_NEW_ICON)) //Checks uniform compatibility
-			uniform_compatible = TRUE
-		if((!human_host.wear_suit) || (human_host.wear_suit.supports_variations & DIGITIGRADE_VARIATION) || !(human_host.wear_suit.body_parts_covered & LEGS) || (human_host.wear_suit.supports_variations & DIGITIGRADE_VARIATION_NO_NEW_ICON)) //Checks suit compatability
-			suit_compatible = TRUE
-
-		if((uniform_compatible && suit_compatible) || (suit_compatible && human_host.wear_suit?.flags_inv & HIDEJUMPSUIT)) //If the uniform is hidden, it doesnt matter if its compatible
-			for(var/obj/item/bodypart/BP as() in human_host.bodyparts)
-				if(BP.bodytype & BODYTYPE_DIGITIGRADE)
-					BP.limb_id = "digitigrade"
-
-		else
-			for(var/obj/item/bodypart/BP as() in human_host.bodyparts)
-				if(BP.bodytype & BODYTYPE_DIGITIGRADE)
-					BP.limb_id = "lizard"
-	///End digi handling
+	// ///Digi handling
```

Comment:
instead of this i'd just remove the code for it we won't be needing it again most likely